scrapers/JaysPov.yml

```diff
@@ -0,0 +1,30 @@
+name: "Jays POV"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - jayspov.net
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="container-fluid video-title"]/h1/text()
+        replace:
+          - regex: '^(.+?)\ \-\ (.+)$'
+            with: $2
+      Details: //meta[@name="og:description"]/@content
+      Date:
+        selector: //div[@class="release-date"]/text()
+        replace:
+          - regex: '^Released\:(.*)$'
+            with: $1
+        parseDate: Jan 2, 2006
+      Image: //meta[@property="og:image"]/@content
+      Studio:
+        Name: //div[@class="studio"]/text()
```

Comment:
the studio is inside a span so you should either use a selector like this
```yaml
Name:
  selector: //div[@class="studio"]
```
to get the whole text and then do a regex replace
or use this
```yaml
Name: //div[@class="studio"]/span[not(contains(.,"Studio:"))]
```